joplin/pages/official_documents_page/factories.py

```diff
@@ -27,14 +14,80 @@ class Meta:
         model = Document
 
 
-class OfficialDocumentPageDocumentFactory(factory.DjangoModelFactory):
+class OfficialDocumentCollectionDocumentFactory(factory.django.DjangoModelFactory):
     page = factory.SubFactory(
-        'official_documents_page.factories.OfficialDocumentsPageFactory',
+        'base_page.factories.JanisBasePageWithTopicsFactory',
```

Comment:
In that case, then should this be `OfficialDocumentPageFactory` instead of `base_page.factories.JanisBasePageWithTopicsFactory`?

Reply:
I think you're right. I'll change it.